src/network/network.rs

```diff
@@ -218,9 +218,12 @@ impl<T> WsConnection<T> {
                         };
                         let JustSaying { subject, body } =
                             t_c!(serde_json::from_value(value[1].take()));
-                        go!(move || if let Err(e) = T::on_message(ws, subject, body) {
-                            error!("{}", e);
-                        });
+                        go!(
+                            move || if let Err(e) = T::on_message(ws.clone(), subject, body) {
+                                let error = json!(e.to_string());
+                                t!(ws.send_error(error));
```

Comment:
should we tell the client about our internal error? JustSaying message doesn't have to need a response.